libbpf-tools/oomkill.c

```diff
@@ -133,6 +136,13 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
+	buf = bpf_buffer__new(obj->maps.events, obj->maps.heap);
+	if (!buf) {
+		err = -errno;
+		warn("failed to create ring/perf buffer: %d\n", err);
```

Comment:
Use `fprintf` for consistency. (`warn` is not defined, no ?)

Reply:
Next time, please address all comments and do NOT ignore compilation warnings.

Reply:
Sorry, I missed this somehow. Should I create a followup PR? It would be nice if the CI could catch these.

Reply:
Yes, please.

Reply:
Please see https://github.com/iovisor/bcc/pull/4294